src/utils/enums.ts

```diff
@@ -343,3 +343,8 @@ export const REDIRECT_FLOW_CONFIG: {
     requiresLogin: true,
   },
 };
+
+export enum SORT_SPL_TOKEN {
+  USD = "usd",
```

Comment:
should rename to be TOKENVALUE or VALUE
usd is misleading as u provide the currency as well